actix-web/src/server.rs

```diff
@@ -189,8 +183,8 @@ where
     ///
     /// By default max connections is set to a 256.
     #[allow(unused_variables)]
+    #[cfg(feature = "__tls")]
     pub fn max_connection_rate(self, num: usize) -> Self {
-        #[cfg(any(feature = "rustls-0_20", feature = "rustls-0_21", feature = "openssl"))]
```

Comment:
moving the flag like this is a breaking change, please move it back and we'll be able to get this merged

Reply:
Moved back to statement level, thanks for review!